CONTRIBUTING.md

```diff
@@ -218,7 +218,7 @@ Server will be run on http://localhost:8080
 Zeppelin manages it's issues in Jira. [https://issues.apache.org/jira/browse/ZEPPELIN](https://issues.apache.org/jira/browse/ZEPPELIN)
 
 ## Where to Start
-You can find issues for [beginner](https://issues.apache.org/jira/browse/ZEPPELIN-924?jql=project%20%3D%20ZEPPELIN%20and%20status%20%3D%20Open%20and%20labels%20in%20\(beginner%2C%20newbie\))
+You can find issues for [beginner](https://issues.apache.org/jira/issues/?jql=labels%20%3D%20beginner%20and%20project%3DZeppelin))
```

Comment:
@zjffdu Thanks for your contribution! There is useless `)` end of the sentence. And is there any particular reason for remove `newbie` tag in this link?